apis/handlers/deleteprog.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Contributors to the L3AF Project.
+// SPDX-License-Identifier: Apache-2.0
+
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"io/ioutil"
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/l3af-project/l3afd/kf"
+	"github.com/l3af-project/l3afd/models"
+)
+
+// DeleteEbpfProgram   remove Ebpf programs on node
+// @Summary removing Ebpf Programs on node
+// @Description Adding new Ebpf Programs on node
+// @Accept  json
+// @Produce  json
+// @Param cfgs body []models.L3afBPFPrograms true "BPF programs"
+// @Success 200
+// @Router /l3af/configs/v1/delete [post]
+func DeleteEbpfProgram(ctx context.Context, kfcfg *kf.NFConfigs) http.HandlerFunc {
```

Comment:
```suggestion
func DeleteEbpfPrograms(ctx context.Context, kfcfg *kf.NFConfigs) http.HandlerFunc {
```
same question here, shouldn't this be plural?